build-tests/api-documenter-test/build.js

```diff
@@ -26,7 +26,7 @@ if (process.argv.indexOf('--production') >= 0) {
 
 // Run the API Documenter command-line
 executeCommand('node node_modules/@microsoft/api-documenter/lib/start '
-  + 'yaml --input-folder etc --output-folder etc/yaml');
+  + 'generate --input-folder etc --output-folder etc/yaml');
```

Comment:
>generate [](start = 5, length = 8)

Should this be an entirely separate test?

Reply:
The new `generate` action is meant to replace the `yaml` action. It's still under experimental title but as we polish it the plan is having just one action `generate` that will look for an `api-documenter.json` config file where we would specify what documenter to be used: yaml vs markdown.